docs/releasing.md

```diff
@@ -116,7 +116,7 @@ include/k4arecord/types.h                                      | :white_check_ma
 linux-ubuntu/x64/release/libdepthengine.so \*                  |                    |                    | :white_check_mark: |                    |
 linux-ubuntu/x64/release/libdepthengine.so.1.0 \*              |                    |                    | :white_check_mark: |                    |
 linux-ubuntu/x64/release/libk4a.so (symlink)                   |                    |                    |                    | :white_check_mark: |
-linux-ubuntu/x64/release/libk4a.so.1.1 (symlink)               |                    |                    |                    | :white_check_mark: |
+linux-ubuntu/x64/release/libk4a.so.1.1 (symlink)               |                    |                    | :white_check_mark: |                   |
```

Comment:
Should we consider changing libk4a.so.1.1 to libk4a.so.1.x so we don't have to update this each time we release a new version?